src/SBP_operators.jl

```diff
@@ -639,7 +639,7 @@ the quadrature rule associated with the SBP derivative operator `D`.
 """
 function integrate(func, u::AbstractVector, D::DerivativeOperator)
     @boundscheck begin
-        length(u) == length(grid(D))
+        length(u) == length(grid(D)) || throw(DimensionMismatch())
```

Comment:
```suggestion
        length(u) == length(grid(D)) || throw(DimensionMismatch("vector should have same size as grid"))
```
Maybe add some informative message?